docs/examples/wasm/webpack/src/base64.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Base64_encoding_and_decoding
+ */
+
+function b64ToUint6(n) {
+    return n > 64 && n < 91  ? n - 65
+         : n > 96 && n < 123 ? n - 71
+         : n > 47 && n < 58  ? n + 4
+         : n === 43 ? 62
+         : n === 47 ? 63
+         : 0
+}
+
+function uint6ToB64(c) {
+    return c < 26 ? c + 65
+         : c < 52 ? c + 71
+         : c < 62 ? c - 4
+         : c === 62 ? 43
+         : c === 63 ? 47
+         : 65
+}
+
+export function Base64ToBytes(sBase64, nBlocksSize) {
```

Comment:
i'm so glad that you didn't copy-pasted code from mozilla's examples with their naming and changed it to readable function names)